examples/swift/fill/01_fill.re

```diff
@@ -0,0 +1,109 @@
+// re2swift $INPUT -o $OUTPUT
+import Foundation
+
+struct Input {
+  static let bufferSize = 4095
+
+  var file: FileHandle
+
+  var buffer   = ContiguousArray<UInt8>(repeating: 0, count: Self.bufferSize + 1)
+  var yylimit  = Self.bufferSize
+  var yycursor = Self.bufferSize
+  var yymarker = Self.bufferSize
+  var token    = -1
+  var eof      = false
+}
+
+extension Input {
+  mutating func lex() -> Int? {
+    var count = 0
+    loop: while true {
+      self.token = self.yycursor
+      /*!re2c
+        re2c:api = record;
+        re2c:eof = 0;
+        re2c:variable:yyrecord = "self";
+        re2c:YYINPUT = "self.buffer";
```

Comment:
Can you rename `buffer` field to `yyinput` and drop this config? Here in other tests.

Reply:
Done for both fill tests.